client/components/collection-list.jsx

```diff
@@ -6,22 +6,43 @@ export default class CollectionList extends React.Component {
     if (!this.props.collections) return null;
     const collectionList = this.props.collections.map((collection, index) => {
       const image = collection.image ? '' : 'hidden';
-      return (
-        <li key={collection.collectionId} className="collectionLi">
-          <div className="collectionImageColumn">
-            <img src={collection.image} alt={`${collection.name} image`} className={`collectionImage ${image}`} />
-          </div>
-          <div className="collectionTextColumn">
-            <h3 className="collectionNameText">
-              {collection.name}
-            </h3>
-            <h3 className="collectionCountText">
-              {collection.count} places saved
-            </h3>
-          </div>
-          <SaveButton saving={this.props.saving} collectionId={collection.collectionId} handleClick={this.props.handleClick} handleSave={this.props.handleSave} />
-        </li>
-      );
+      if (this.props.saving) {
+        return (
+          <li className="collectionLi" key={collection.collectionId}>
+            <div className="collectionImageColumn">
+              <img src={collection.image} alt={`${collection.name} image`} className={`collectionImage ${image}`} />
+            </div>
+            <div className="collectionTextColumn">
+              <h3 className="collectionNameText">
+                {collection.name}
+              </h3>
+              <h3 className="collectionCountText">
+                {collection.count} places saved
+              </h3>
+            </div>
+            <SaveButton saving={this.props.saving} collectionId={collection.collectionId} handleClick={this.props.handleClick} handleSave={this.props.handleSave} />
+          </li>
+        );
+      } else {
+        return (
+          <a href={`#collection?collectionId=${collection.collectionId}`} key={collection.collectionId} >
```

Comment:
Instead of conditionally wrapping the whole thing in an anchor tag, swap out the SaveButton for an anchor styled like the save button


```jsx
<li>
  <other-stuff/>
  { this.props.saving
     ? <SaveButton etc etc />
     : <a href=...>View</a>
  }
</li>
```